docs/source/guides/libraries.rst

```diff
@@ -67,12 +67,256 @@ Python library
 Where to get
 ^^^^^^^^^^^^
 
+Prerequirements:
```

Comment:
I doubt that for PIP version people really need cmake, boost, swig, protobuf. Please tell why they need that

Reply:
the installation process via pip is almost the same as from terminal. It clones repo and compiles it. That is why you need all of this even with pip

Reply:
boost is a header only dependency, and still we need it?